dart/dynamics/Shape.h

```diff
@@ -121,7 +126,7 @@ class Shape : public virtual common::Subject
   //           biased mesh shape. Two Vector3ds might be better; one is for
   //           minimum verterx, and the other is for maximum verterx of the
   //           bounding box.
-  const Eigen::Vector3d& getBoundingBoxDim() const;
+  Eigen::Vector3d getBoundingBoxDim() const;
```

Comment:
This function should be marked as `DEPRECATED` and the `TODO` should probably be made a comment that points the user to `getBoundingBox`.